framework/src/components/storage/entities/account.js

```diff
@@ -48,6 +48,7 @@ class Account extends BaseEntity {
 			booleanToInt,
 		);
 		this.addField('balance', 'string', { filter: ft.NUMBER });
+		this.addField('nonce', 'string', { filter: ft.BOOLEAN });
```

Comment:
boolean? its number know?

Reply:
Value is string, filter type is `BOOLEAN`. Which will result in `nonce_eql` and `nonce_ne` filters.